javascript/gdx_getting_started_2.html

```diff
@@ -1,3 +1,15 @@
+  <!--
+  This example uses the GDX JavaScript library to collect all of the sensor information from a Vernier GoDirect Sensor. 
+  Press the 'Select GoDirect Device' button to open communication to the device in a Chrome browser. 
+  This example collects 10 data points and disconnects.
+  
+  This program uses the following functions from the GDX Library:
+  
+  gdxDevice.selectDevice();
+  gdxDevice.close();
+  gdxDevice.sensors.filter();
+  -->  
+
```

Comment:
Perhaps give this a similar title to the others. Like: For example 2: Simple Data Collection